src/Microsoft.AspNetCore.Mvc.Abstractions/Filters/ExceptionContext.cs

```diff
@@ -69,6 +69,11 @@ public virtual ExceptionDispatchInfo ExceptionDispatchInfo
         }
 
         /// <summary>
+        /// Gets or sets an indication that the <see cref="Exception"/> has been handled.
+        /// </summary>
+        public virtual bool ExceptionHandled { get; set; } = false;
```

Comment:
Why virtual? I don't think there's an easy way to replace the exception context with your own, and being get set I don't see how making it virtual would improve unit testing which is the other scenario I would consider. @rynowak am I missing something?